techniques/silverlight/SL1.html

```diff
@@ -77,7 +77,7 @@ <h3>Changing AudioStreamIndex </h3>
     &lt;/Grid&gt;
 </pre>
             <p>The following is the C# logic. </p>
-            <pre xml:space="preserve">        private void AltAudioBtn_Click(object sender, RoutedEventArgs e)
+            <pre>        private void AltAudioBtn_Click(object sender, RoutedEventArgs e)
```

Comment:
```suggestion
            <pre><code>        private void AltAudioBtn_Click(object sender, RoutedEventArgs e)
```

and add matching `</code>`